server.js

```diff
@@ -19,11 +19,13 @@ function start(processId) {
   // app modules
   const app = require('./app');
 
+  // Setup Gateway client.
+  require('./lib/gateway').getClient();
```

Comment:
You'd think so... but there seems to be an order of operations issue for the first `GET /campaigns/:id` request, and I think it may be because calling`getNewInstance` doesn't necessarily mean we have an access token to make the Rogue API call.


```
11:04:40 AM web.1 |  debug: Campaign cache miss 
11:04:40 AM web.1 |  { campaignId: 7 }
11:04:40 AM web.1 |  error: Cannot read property 'access_token' of undefined
11:04:40 AM web.1 |  verbose:  
11:04:40 AM web.1 |  { error: 
11:04:40 AM web.1 |     { code: 500,
11:04:40 AM web.1 |       message: 'Cannot read property \'access_token\' of undefined' } }
11:04:40 AM web.1 |  info: ClientCredentials.setup: token has been set
```

Reply:
Ohhh, interesting. Might be nice to add a `TODO` or comment here explaining that in a future PR, since I could totally imagine coming across this a few months down the line and thinking it was safe to delete.